rdf-js-tests.ts

```diff
@@ -84,6 +84,21 @@ function test_datafactory() {
 
     const variable: Variable = dataFactory.variable ? dataFactory.variable('v1') : <any> {};
 
+    const defaultGraph: DefaultGraph = dataFactory.defaultGraph();
+
+    const term1: Term = dataFactory.fromTerm(dataFactory.namedNode(""));
+    const term2: Term = dataFactory.fromTerm(dataFactory.blankNode());
+    const term3: Term = dataFactory.fromTerm(dataFactory.literal(""));
+    const term4: Term = dataFactory.fromTerm(dataFactory.variable ? dataFactory.variable("v1") : <any> {});
+    const term5: Term = dataFactory.fromTerm(dataFactory.defaultGraph());
```

Comment:
```suggestion
    const term1: NamedNode = dataFactory.fromTerm(dataFactory.namedNode(""));
    const term2: BlankNode = dataFactory.fromTerm(dataFactory.blankNode());
    const term3: Literal = dataFactory.fromTerm(dataFactory.literal(""));
    const term4: Variable = dataFactory.fromTerm(dataFactory.variable ? dataFactory.variable("v1") : <any> {});
    const term5: DefaultGraph = dataFactory.fromTerm(dataFactory.defaultGraph());
```

Reply:
I updated this suggestion too, to match the expectation that returned types are the base interfaces